charts/kube-prometheus-stack/templates/exporters/kubelet/servicemonitor.yaml

```diff
@@ -120,10 +122,7 @@ spec:
     honorLabels: {{ .Values.kubelet.serviceMonitor.honorLabels }}
     honorTimestamps: {{ .Values.kubelet.serviceMonitor.honorTimestamps }}
     trackTimestampsStaleness: {{ .Values.kubelet.serviceMonitor.trackTimestampsStaleness }}
```

Comment:
After reading https://github.com/prometheus-community/helm-charts/pull/5050#issuecomment-2543009534, does it make sense to remove `honorTimestamps` and `trackTimestampsStaleness` from non-cadvisor endpoints?

Reply:
I was going to do that as a separate change after this refactor.